GetMessage/handler.ts

```diff
@@ -134,9 +131,6 @@ type IGetMessageHandler = (
   | IResponseErrorInternal
 >;
 
-const LegalMessagePattern = t.interface({ legal_data: LegalData });
```

Comment:
Ditto. We will allow for legal data retrieval for the same reason we left the specs unchanged.

Reply:
[043f255](https://github.com/pagopa/io-functions-services/pull/278/commits/043f2559321c790c2627335f9e21ad971a52de5b)